src/main/resources/regression/car17.yaml

```diff
@@ -0,0 +1,97 @@
+---
+name: car17
+index_command: target/appassembler/bin/IndexCollection
+index_utils_command: target/appassembler/bin/IndexUtils
+search_command: target/appassembler/bin/SearchCollection
+topic_root: src/main/resources/topics-and-qrels/
+qrels_root: src/main/resources/topics-and-qrels/
+index_root:
+ranking_root:
+generator: LuceneDocumentGenerator
+threads: 40
+index_options:
+  - -storePositions
+  - -storeDocvectors
```

Comment:
We've decided to enable `-storeRawDocs` for regression test. 
Could you please add it?